src/Http/MiddlewareQueue.php

```diff
@@ -34,11 +36,11 @@ class MiddlewareQueue implements Countable
     protected $queue = [];
 
     /**
-     * The queue of middleware callables.
+     * The queue of middlewares.
      *
-     * @var callable[]
+     * @var \Psr\Http\Server\MiddlewareInterface[]
      */
-    protected $callables = [];
+    protected $middlewares = [];
```

Comment:
I learned this in my PR https://github.com/cakephp/app/pull/630:

> Plural of middleware is middleware.

Reply:
I think it's acceptable to ignore grammatical rules for a variable name to make it apparent that this is a list of multiple middleware.

Also there are some [source](https://en.wiktionary.org/wiki/middleware) which state that in technical context the plural form is `middlewares`.